Formula/m/morpheus.rb

```diff
@@ -2,8 +2,9 @@ class Morpheus < Formula
   desc "Modeling environment for multi-cellular systems biology"
   homepage "https://morpheus.gitlab.io/"
   url "https://gitlab.com/morpheus.lab/morpheus/-/archive/v2.3.9/morpheus-v2.3.9.tar.gz"
-  sha256 "2c948b6537dfc09b3a7fe536c722a6effbf0dbad30341f0aec0635a2806bd0f8"
+  sha256 "d27b7c2b5ecf503fd11777b3a75d4658a6926bfd9ae78ef97abf5e9540a6fb29"
```

Comment:
How did it change without the download changing? Has upstream been compromised?

Reply:
Is there any reference to that in the upstream repo? It would be good to know for other people who might have a similar issue.

Reply:
I added a [note](https://gitlab.com/morpheus.lab/morpheus/-/releases/v2.3.9) upstream to make the checksum change transparent there as well.